src/AltinnCore/Common/Services/Implementation/AuthenticationAppSI.cs

```diff
@@ -56,6 +56,7 @@ public async Task<HttpResponseMessage> RefreshToken()
             if (response.StatusCode == System.Net.HttpStatusCode.OK)
             {
                 string refreshedToken = GetCookieValueFromResponse(response, Constants.General.RuntimeCookieName);
+                _logger.LogInformation($"refreshedtoken {refreshedToken}");
```

Comment:
Do not log the token or session cookie. 

7.1.1 Verify that the application does not log credentials or payment details. Session tokens should only be stored in logs in an irreversible, hashed form. (C9, C10)

Reply:
Removed logging now